DXMainClient/DXGUI/Multiplayer/CnCNet/LoadSaveGameOptionsWindow.cs

```diff
@@ -0,0 +1,245 @@
+        {
+            if (IsCreatePresetSelected)
+            {
+                // show the field to specify a new name when "create" option is selected in drop down
+                tbNewPresetName.Enable();
+                lblNewPresetName.Enable();
+            }
+            else
+            {
+                // hide the field to specify a new name when an existing preset is selected
+                tbNewPresetName.Disable();
+                lblNewPresetName.Disable();
+            }
+        }
+
+        /// <summary>
+        /// Refresh the state of the load/save button
+        /// </summary>
+        private void RefreshLoadSaveBtn()
+        {
+            if (_isLoad)
+                btnLoadSave.Enabled = !IsSelectPresetSelected;
+            else
+                btnLoadSave.Enabled = !IsCreatePresetSelected || !IsNewPresetNameFieldEmpty;
+        }
+
+        private bool IsCreatePresetSelected => ddPresetSelect.SelectedItem == ddiCreatePresetItem;
+        private bool IsSelectPresetSelected => ddPresetSelect.SelectedItem == ddiSelectPresetItem;
+        private bool IsNewPresetNameFieldEmpty => string.IsNullOrWhiteSpace(tbNewPresetName.Text);
+
+        /// <summary>
+        /// Populate the preset drop down from saved presets
+        /// </summary>
+        private void LoadPresets()
+        {
+            ddPresetSelect.Items.Clear();
+            ddPresetSelect.Items.Add(_isLoad ? ddiSelectPresetItem : ddiCreatePresetItem);
+            ddPresetSelect.SelectedIndex = 0;
+
+            ddPresetSelect.Items.AddRange(GameOptionPresets.Instance
+                .GetPresetNames()
+                .OrderBy(name => name)
+                .Select(name => new XNADropDownItem()
+                {
+                    Text = name
+                }));
+        }
+
+        /// <summary>
+        /// Show the current window in the "load" mode context
+        /// </summary>
+        private void ShowLoad()
+        {
+            LoadPresets();
+
+            // do not show fields to specify a preset name during "load" mode
+            lblNewPresetName.Disable();
+            tbNewPresetName.Disable();
+        }
+
+        /// <summary>
+        /// Show the current window in the "save" mode context
+        /// </summary>
+        private void ShowSave()
+        {
+            LoadPresets();
+
+            // show fields to specify a preset name during "save" mode
+            lblNewPresetName.Enable();
+            tbNewPresetName.Enable();
+        }
+
+        private void BtnLoadSave_LeftClick(object sender, EventArgs e)
+        {
+            var selectedItem = ddPresetSelect.Items[ddPresetSelect.SelectedIndex];
+            if (_isLoad)
+                PresetLoaded?.Invoke(this, selectedItem.Text);
+            else
+            {
+                var presetName = IsCreatePresetSelected ? tbNewPresetName.Text : selectedItem.Text;
+                PresetSaved?.Invoke(this, presetName);
+            }
```

Comment:
```suggestion
            if (_isLoad)
            {
                PresetLoaded?.Invoke(this, selectedItem.Text);
            }
            else
            {
                var presetName = IsCreatePresetSelected ? tbNewPresetName.Text : selectedItem.Text;
                PresetSaved?.Invoke(this, presetName);
            }
```

Reply:
Let's keep braces/no braces consistent in range of the same branching construction.